db/R/get.trait.data.R

```diff
@@ -272,15 +272,16 @@ runModule.get.trait.data <- function(settings) {
     modeltype <- settings$model$type
     dbfiles <- settings$database$dbfiles
     database <- settings$database$bety
-    forceupdate <- !is.null(settings$meta.analysis) & settings$meta.analysis$update
+    if ((forceupdate == 'AUTO') || !as.logical(forceupdate))
+    forceupdate <- !is.null(settings$meta.analysis) && (as.logical(settings$meta.analysis$update) || (settings$meta.analysis$update == 'AUTO'))
     settings$pfts <- get.trait.data(pfts, modeltype, dbfiles, database, forceupdate)
     return(settings)
   } else if(is.Settings(settings)) {
     pfts <- settings$pfts
     modeltype <- settings$model$type
     dbfiles <- settings$database$dbfiles
     database <- settings$database$bety
-    forceupdate <- !is.null(settings$meta.analysis) &settings$meta.analysis$update
+    forceupdate <- !is.null(settings$meta.analysis) && (as.logical(settings$meta.analysis$update) || (settings$meta.analysis$update == 'AUTO'))
```

Comment:
hmm that's correct, but I'm a bit confused. The web interface created initial pecan.xml doesn't have settings$meta.analysis$update but pecan.METProcess.xml does, so we end up not running that if-chunk because forceupdate is set to true, which causes error later in the workflow